npc/re/kafras/Zonda_Teleportation_Services.txt

```diff
@@ -0,0 +1,95 @@
+//===== rAthena Script ======================================= 
+//= Zonda Teleportation Services
+//===== Changelogs: ==========================================
+//= 1.0 First Version. [JohnnyPlayy]
+//===== Description: ========================================= 
+// Texts from this script were translated according to KRO
+//============================================================
+
+-	script	ZondaTeleportationServices	HIDDEN_NPC,{
+	cutin "zonda_do01", 2;
+	mes "[Zonda Employee]";
+	mes "Welcome.";
+	mes "Our team is always working to exceed your expectations regarding quality of service.";
+	mes "Please tell me your destiny?";
+	next;
+	switch( select( "North Bifrost", "Eclage", "Splendide", "Midgard Expedition Camp", "Manuk", "Outskirts of Kamidal Mountain", "Cancel.") ) {
+	case 1:
+		callsub( S_Zone, "North Bifrost", 17000,850, 7213 ); // Teleport value not confirmed
+		warp "bif_fild02",160,230; // Unverified Coordinates
+		end;
+	case 2:
+		callsub( S_Zone, "Eclage", 17000,850, 11310 );
+		warp "eclage",111,39;
+		end;
+	case 3:
+		callsub( S_Zone, "Splendide", 17000,850, 10078 );
+		warp "splendide",171,115;
+		end;
+	case 4:
+		callsub( S_Zone, "Midgard Expedition Camp", 19000,950, 10078 );
+		warp "mid_camp",210,288;
+		end;
+	case 5:
+		callsub( S_Zone, "Manuk", 22000,1100, 10078 );
+		warp "manuk",263,177;
+		end;
+	case 6:
+		callsub( S_Zone, "Outskirts of Kamidal Mountain", 24000,1200, 7184 );
+		warp "dic_fild01",188,187;
+		end;
+	case 7:
+		mes "[Zonda Employee]";
+		mes "We are always striving to ensure the best services for our customers.";
+		mes "Thank you for using our services.";
+		close3;
+	}
+
+S_Zone:
+	.@zeny_req = getarg(1);
+	.@item_req = getarg(2);
+	.@quest_req = getarg(3);
+	
+	if (isbegin_quest(.@quest_req) == 0) {
+		mes "[Zonda Employee]";
+		//mes getarg(0);
+		mes "You must complete the access mission to teleport to " + getarg(0) + " before using our Teleportation services.";
+		close3;
+		cutin "zonda_do01", 255;
```

Comment:
```suggestion
```

Reply:
All the speeches and functions of these NPCs I collected. 
I even tested these NPCs with other characters to make sure I hadn't forgotten any text or special function.
But a mission I didn't complete that and line 19.
Line 55 that is disabled and really official at KRO, but I disabled this line because it didn't make me feel!